### DISCUSSION TITLE
The SV number selected for control does not respond on the Fuji thermostat.

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1683] [General]
In past versions can be set to switch normally.

**Comment by Terracotta-6** [comment]


**Reply by Terracotta-6** [reply]
I don't know the background of this development project, try to use.

**Comment by MAKOMO** [comment]
I think I catched this one. A typo was introduced in 2.8.4 causing this function to fail. I commited a fix in cf09ec42a92383d2ce31ed60ac6c089052357259 and hope this will resolve the issue. A new continuous build should soon be available.

**Reply by Terracotta-6** [reply]
This has been fixed. Thank You！

**Comment by Terracotta-6** [comment]


**Reply by MAKOMO** [reply]
You need to have the Ramp/Soak OFF and the PID ON.

**Comment by Terracotta-6** [comment]
![92167bfcd5ac58288961d9b1bc328b8](https://github.com/user-attachments/assets/675f3451-b46a-4991-acd1-39c831f5a886)
Wrong command ！

**Reply by Terracotta-6** [reply]
Switchover between RUN and standby
Switchover the operation mode between RUN and standby

**Comment by MAKOMO** [comment]
Well, that whole Fuji PID support was written over 10 years ago, before Artisan supported the general MODBUS protocol. Maybe all this should just be removed.

**Reply by Terracotta-6** [reply]
Fuji Temperature Control General MODBUS protocol for more flexible configuration control, Test it out of curiosity，Wasting your precious time.
